spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index Page' do
+  let!(:user1) { User.create(name: 'Ajrass', photo: 'https://avatars.githubusercontent.com/u/130588108?v=4') }
+  let!(:user2) { User.create(name: 'Khadija', photo: 'https://secure.b8cdn.com/images/uploads/user_photos/67/56263867_20211019183159.jpg') }
+  let!(:post1) { Post.create(author: user1, title: 'Post 01') }
+  let!(:post2) { Post.create(author: user2, title: 'Post 02') }
+
+  scenario 'Displays usernames' do
+    visit users_path
+    expect(page).to have_content(user1.name)
+    expect(page).to have_content(user2.name)
+  end
+
```

Comment:
- [ ] All of the integration tests are failing, Please make sure to fix this and all tests should pass without any errors..

![image](https://github.com/Young-Gaius/Rails-Blog-App/assets/83803414/809f74c6-ebbe-474f-a5b1-23c4778884c7)